server/gendb/interface.go.tmpl

```diff
@@ -136,6 +153,18 @@ type Get{{ $modelName }}sBy{{ $indexName }}Input struct {
   DisableConsistentRead bool
   // Limit is an optional limit of how many items to evaluate.
 	Limit  *int64
+  {{- if gt $lenNonIndexAttributes 0 }}
+    // Filters is an optional array of filters to apply on various table attributes
+    FilterValues []{{ $modelName }}By{{ $indexName }}FilterValues
+    // FilterExpression is the filter expression to be applied to our fitlered attributes 
+    // when referencing an attribute use #ATTRIBUTE_NAME
+    // ex: if the attribute is called "created_at" in its wag definition use #CREATED_AT
+    // when referencing one of the given values use :{attribute_name}_value0, :{attribute_name}_value1, etc.
+    // ex: if the attribute is called "created_at" in its wag definition use :created_at_value0, created_at_value1, etc.
```

Comment:
How come here we are using the lower case name but on line 160 we use the upper case? I tried to look at the link below but didn't find the answer

Reply:
it's just a naming convention that I've chosen to be similar to the naming convention used already for the hash/range keys.  it doesn't necessarily have to be this way.  See https://github.com/Clever/wag/blob/e2e3ba730ab6ec7243fbbc5b1a860c70eb973aa3/samples/gen-go-db/server/db/dynamodb/thing.go#L324 for an example of how its already used for keys